samples/deployment/aws/deploy_and_destroy.yaml

```diff
@@ -0,0 +1,130 @@
+- name: "My AWS infrastructure"
```

Comment:
Let's remove the pronoun `my` and write a better description explaining what this playbook does.

Reply:
fixed